apps/aecore/src/aec_keys.erl

```diff
@@ -511,12 +512,19 @@ p_gen_new_peer(Pwd, PubFile, PrivFile) ->
 ensure_dir(KeysDir) ->
     case filelib:is_dir(KeysDir) of
         false ->
-            KeysDirFile = filename:join(KeysDir, "keyfile"),
-            ok = filelib:ensure_dir(KeysDirFile);
+            case ensure_dir_(KeysDir) of
+                ok  -> ok;
+                Err -> error({could_not_ensure_key_dir, KeysDir, Err})
+            end;
         true ->
             ok
     end.
 
+%% Note: `filelib:ensure_path/1` was introduced in OTP-25
```

Comment:
how is this comment related 🤔

Reply:
It is just an explanation for the silly decision to use `ensure_dir` when `ensure_path` is what we want...